aiida_pseudo/cli/install.py

```diff
@@ -104,13 +105,30 @@ def download_sssp(
     :param filepath_archive: absolute filepath to write the pseudopotential archive to.
     :param filepath_metadata: absolute filepath to write the metadata file to.
     :param traceback: boolean, if true, print the traceback when an exception occurs.
+    :return: Latest patch version of the requested minor version
```

Comment:
Now just need to update the return type of this function as well and than this is good to merge

Reply:
Righto! Missed that one, thanks

Reply:
More a sign that we should probably at some point at `mypy` to the pre-commit. On the one hand I like added typing, but on the other hand `mypy` has been given so much trouble that sometimes it annoys me to no end. :rock: me :hardplace: